ui/src/notebooks/components/Notebook.tsx

```diff
@@ -18,27 +23,33 @@ import 'src/notebooks/style.scss'
 
 const NotebookPage: FC = () => {
   return (
-    <CurrentNotebook>
-      <ResultsProvider>
-        <RefProvider>
-          <ScrollProvider>
-            <Page titleTag="Flows">
-              <NotebookHeader />
-              <Page.Contents
-                fullWidth={true}
-                scrollable={false}
-                className="notebook-page"
-              >
-                <div className="notebook">
-                  <MiniMap />
-                  <PipeList />
-                </div>
-              </Page.Contents>
-            </Page>
-          </ScrollProvider>
-        </RefProvider>
-      </ResultsProvider>
-    </CurrentNotebook>
+    <AppSettingProvider>
```

Comment:
I have no context on this. I think @alexpaxton popped this in earlier in the branch for a specific reason